modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/common/services/notification.ts

```diff
@@ -16,9 +16,13 @@ import {axios} from './liferay/api';
 
 const headlessAPI = 'o/headless-user-notification/v1.0';
 
-export function getUserNotification(pageSize: number, page: number) {
+export function getUserNotification(
+	pageSize: number,
+	page: number,
+	order: string
+) {
 	return axios.get(
-		`${headlessAPI}/my-user-notifications/?&pageSize=${pageSize}&page=${page}`
+		`${headlessAPI}/my-user-notifications/?&pageSize=${pageSize}&page=${page}&sort=dateCreated:${order}`
 	);
 }
```

Comment:
The "boy-scout rule': 'Always leave the code better than you found it. '

Reply:
Done!